rpc_state_reader_sn_api/src/lib.rs

```diff
@@ -1260,10 +1433,24 @@ mod starknet_in_rust_transaction_tests {
                     .as_ref()
                     .unwrap()
                     .internal_calls
-                    .len()
+                    .len(),
+                "internal calls length mismatch"
             );
 
-            assert_eq!(actual_fee, receipt.actual_fee);
+            let diff = receipt.actual_fee.abs_diff(actual_fee);
+
+            if diff > 0 {
+                let upper_value = receipt.actual_fee.max(diff);
+                let lower_value = receipt.actual_fee.min(diff);
+                let diff_percent = (100 * lower_value + upper_value / 2) / upper_value;
+
+                if diff_percent > 5 {
```

Comment:
Careful with this, the quotient will be truncated since we're operating with integers. This means that you're probably checking whether `real_diff_percent >= 6.0`.